tests/test_harness_capability_profiles.py

```
@@ -0,0 +1,430 @@
+        summarize = self.run_manager(root, "summarize", "--write", "--json")
+        self.assertEqual(summarize.returncode, 0, summarize.stderr)
+
+    def run_manager(self, root: Path, *args: str) -> subprocess.CompletedProcess[str]:
+        return subprocess.run(
+            [sys.executable, str(MANAGER), "--root", str(root), *args],
+            cwd=REPO_ROOT,
+            check=False,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            text=True,
+        )
```

Ruff (0.15.12)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)